Sandbox/BubbleSortLayer.cpp

```diff
@@ -63,7 +66,13 @@ void BubbleSortLayer::onUpdate(Aquarius::timeDelta_t ts)
         resetSort = false;
     }
 
-    if (isActive())
+    if(pauseSort)
+    {
+        Aquarius::Renderer::Clear();
```

Comment:
I dont know why you need multiple different clear calls in the same onUpdate method